src/Core/src/AwsError/AwsErrorFactoryFromResponseTrait.php

```diff
@@ -4,9 +4,6 @@
 
 use Symfony\Contracts\HttpClient\ResponseInterface;
 
-/**
- * @internal
```

Comment:
I made the trait non-final because it is the only implementation of `createFromResponse` and so it might be useful for #1485 when implementing an error factory internally.

Alternatively, we could remove `createFromResponse` from the interface, inlining the extraction of the `createFromContent` arguments. A class with a custom logic in `createFromContent` is incompatible with the chain factory anyway because the chain factory performs this extraction on its own and then calls only `createFromContent`